### PR TITLE
Function alignment

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

File: core/arch/arm/sm/sm_a32.S [hunk 1]
```diff
@@ -295,9 +295,11 @@ sm_vect_table_a15:
 	write_iciallu
 	isb
 	b	1f
+END_FUNC sm_vect_table_a15
+
```
Comment (etienne-lms) [inline core/arch/arm/sm/sm_a32.S:299]: remove empty line

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@linaro.org>`

**Review by jforissier — COMMENTED** [review]
For "Add alignment parameter to FUNC() and LOCAL_FUNC() macros":

`Reviewed-by: Jerome Forissier <jerome@forissier.org>`

**Review by jforissier — COMMENTED** [review]
For "core: correct aliment for exception vectors":

LGTM but I think the .align -> .balign changes should go in a separate commit, and `.cantunwind` too.

**Review by etienne-lms — COMMENTED** [review]
log header line for commit "core: correct aliment for exception vectors": s/aliment/alignment/

**Comment by jenswi-linaro** [comment]
Updated as requested and tags applied where applicable.

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`

**Comment by jenswi-linaro** [comment]
Tags applied

File: core/arch/arm/sm/sm_a32.S (hunk 1 above)
Comment (jenswi-linaro) [reply core/arch/arm/sm/sm_a32.S:299]: I'm sorry, it seems I missed this comment.

Comment (etienne-lms) [reply core/arch/arm/sm/sm_a32.S:299]: I guess that not a big issue :)

**Pull request merged**